src/builtin/filters/array.ts

```diff
@@ -18,6 +19,41 @@ export function sort<T> (this: FilterImpl, arr: T[], property?: string) {
   })
 }
 
+type OperationType = 'global' | 'scoped'
+function getFromExpression (originalString: string, cb: (value: string, operationType: OperationType) => string, expectedClose?: string, iter = 0): string {
```

Comment:
Interesting that we can evolve the liquidjs script capabilities towards our own needs now.